src/test/java/codeu/controller/RegisterServletTest.java

```diff
@@ -73,10 +75,15 @@ public void testDoPost_NewUser() throws IOException, ServletException {
 
     ArgumentCaptor<User> userArgumentCaptor = ArgumentCaptor.forClass(User.class);
 
-    Mockito.verify(mockUserStore).addUser(userArgumentCaptor.capture());
-    Assert.assertEquals(userArgumentCaptor.getValue().getName(), "Testertest");
+    Mockito.verify(mockUserStore).addUser(userArgumentCaptor.capture()); 
+    User storedUser = userArgumentCaptor.getValue();
+    Assert.assertEquals(storedUser.getName(), "Testertest");
 
+    //BCrypt.checkpw() checks that the second param is correctly hashed  
+    Assert.assertTrue(BCrypt.checkpw("password", storedUser.getPassword()));   
+    Assert.assertTrue(!(storedUser.getPassword().equals("password")));
```

Comment:
`Assert.assertNotEquals("password", storedUser.getPassword())` will do the same thing and give you a clearer error message if the test fails. :) (You can also use `Assert.assertFalse` if you ever have to check a bare boolean.)

It might also be a good idea to add a comment for this assert too, saying that this checks that the stored password is hashed and not plaintext.